src/external/governance/Governor.sol

```diff
@@ -152,7 +153,64 @@ contract Governor is ERC165, IGovernor, AccessControlUpgradeable {
         onlyRole(COMMUNITY_MULTISIG_ROLE)
         validAddress(newFeeManager)
     {
-        feeManager = newFeeManager;
+        feeManager = IFeeManager(newFeeManager);
+    }
+
+    /// @inheritdoc IGovernor
+    function setFeeManagerDefaultProtocolTreasury(
+        address _defaultProtocolTreasury
+    ) external onlyRole(COMMUNITY_MULTISIG_ROLE) {
+        feeManager.setDefaultProtocolTreasury(_defaultProtocolTreasury);
+    }
+
+    /// @inheritdoc IGovernor
+    function setFeeManagerWorkflowTreasuries(address workflow, address treasury)
+        external
+        onlyRole(COMMUNITY_MULTISIG_ROLE)
+    {
+        feeManager.setWorkflowTreasuries(workflow, treasury);
+    }
+
+    /// @inheritdoc IGovernor
+    function setFeeManagerDefaultCollateralFee(uint _defaultCollateralFee)
+        external
+        onlyRole(COMMUNITY_MULTISIG_ROLE)
+    {
+        feeManager.setDefaultCollateralFee(_defaultCollateralFee);
+    }
+
+    /// @inheritdoc IGovernor
+    function setFeeManagerDefaultIssuanceFee(uint _defaultIssuanceFee)
+        external
+        onlyRole(COMMUNITY_MULTISIG_ROLE)
+    {
+        feeManager.setDefaultIssuanceFee(_defaultIssuanceFee);
+    }
+
+    /// @inheritdoc IGovernor
+    function setFeeManagerCollateralWorkflowFee(
+        address workflow,
+        address module,
+        bytes4 functionSelector,
+        bool set,
+        uint fee
+    ) external onlyRole(COMMUNITY_MULTISIG_ROLE) {
+        feeManager.setCollateralWorkflowFee(
+            workflow, module, functionSelector, set, fee
+        );
+    }
+
+    /// @inheritdoc IGovernor
+    function setFeeManagerIssuanceWorkflowFee(
+        address workflow,
+        address module,
+        bytes4 functionSelector,
+        bool set,
+        uint fee
+    ) external onlyRole(COMMUNITY_MULTISIG_ROLE) {
+        feeManager.setIssuanceWorkflowFee(
+            workflow, module, functionSelector, set, fee
+        );
```

Comment:
I will double-check this with the business side, as I can see arguments for both sides, but I assume that these might be functions that both multisigs, but especially the `TEAM_MULTISIG_ROLE`, should be able to access on its own. This is something that will (or at least can) occur pretty often (i.e. whenever a new workflow is deployed that we might have a custom deal with) and I don't see us asking the community multisig all the time to confirm a new fee value.

Maybe (just spitballing) a good solution here would be that the community multisig can set certain boundaries for the team multisig (i.e., such that we can set the fee to a value from 0-5%, but community multisig can do 0-100%) so that we can't just silently set the fee to a super high value without community confirmation.

If that takes too long for now I would probably just:
* Set it to allow any multisig for now
* Revisit this after the first audit handover (i.e. in a week) with an update, where we introduce this - and have it covered in the subsequent audits/contests

What do you think @FHieser?